rosdep/base.yaml

```diff
@@ -2389,6 +2389,11 @@ libpqxx:
     trusty: [libpqxx-4.0]
     utopic: [libpqxx-4.0]
     vivid: [libpqxx-4.0]
+    wily: [libpqxx-4.0]
+    xenial: [libpqxx-4.0]
+    yakkety: [libpqxx-4.0v5]
+    zesty: [libpqxx-4.0v5]
+    artful: [libpqxx-4.0v5]
```

Comment:
I've made the change to allow the merge, but order is an odd constraint for a hash.